FlowCrypt/Controllers/Inbox/InboxRenderable.swift

```diff
@@ -65,4 +63,32 @@ extension InboxRenderable {
         self.date = date ?? Date()
         self.wrappedType = .thread(thread)
     }
+
+    private static func messageTitle(with thread: MessageThread, and folderPath: String) -> String {
+        guard let myEmail = DataService.shared.email else { return "" }
```

Comment:
I feel like this should maybe throw (if we cannot figure out our own email, something is badly wrong)

Reply:
I don't think it worth to handle, but we can if you think it is. ( we need to do that over all app ).

for example, message initialize method will become to throw, and it can leads to other code changes.

do you want me to work on it in scope of this task?

Reply:
I don't think it worth to handle, but we can if you think it is. ( we need to do that over all app ).

for example, message initialize method will become to throw, and it can leads to other code changes.

do you want me to work on it in scope of this task?

Reply:
I think we should create separate task for handling this case (as the same code is used on other app screens too), as well as for empty `context.folderPath`.
Then this PR won't become too big and will be more specific

Reply:
mentioned here https://github.com/FlowCrypt/flowcrypt-ios/issues/863